src/components/DoubleInput/DoubleInput.js

```diff
@@ -28,45 +32,63 @@ class DoubleInput extends PureComponent<Props> {
       onClose,
       valueLeft,
       valueRight,
+      warningMessage,
       placeholderLeft,
       placeholderRight,
       errorMessage,
-      isLoading,
+      isLoadingLeft,
+      isLoadingRight,
     } = this.props
     return (
       <div className='double-input'>
-        <div className='field'>
-          <JInput
-            onChange={onChangeLeft}
-            value={valueLeft}
-            name={placeholderLeft}
-            errorMessage={errorMessage}
-            placeholder={placeholderLeft}
-            type='text'
-            color='gray'
-            sideBorderRadius='left'
-          />
+        <div className='wrap'>
+          <div className='field'>
+            <JInput
+              onChange={onChangeLeft}
+              value={valueLeft}
+              name={placeholderLeft}
+              placeholder={placeholderLeft}
+              type='text'
+              color='gray'
+              sideBorderRadius='left'
+            />
+            {isLoadingLeft && (
+              <div className='loader'>
+                <JLoader color='blue' />
+              </div>
+            )}
+          </div>
+          <div className='field'>
+            <JInput
+              onChange={onChangeRight}
+              value={valueRight}
+              name={placeholderRight}
+              placeholder={placeholderRight}
+              type='text'
+              color='gray'
+              sideBorderRadius='top'
+            />
+            {isLoadingRight && (
+              <div className='loader'>
+                <JLoader color='blue' />
+              </div>
+            )}
+          </div>
         </div>
-        <div className='field'>
-          <JInput
-            onChange={onChangeRight}
-            value={valueRight}
-            name={placeholderRight}
-            placeholder={placeholderRight}
-            type='text'
-            color='gray'
-            sideBorderRadius='top'
-          />
-          {isLoading && (
-            <div className='loader'>
-              <JLoader color='blue' />
-            </div>
-          )}
-        </div>
-        {!onClose &&
+        {onClose &&
         <div className='close' onClick={onClose}>
           <JIcon name='padding-cross' size='medium' color='gray' />
         </div>}
+        {errorMessage &&
+        <div className='message'>
+          <JText value={errorMessage} color='red' size='small' />
+        </div>
+        }
+        {warningMessage &&
```

Comment:
I think, error has biggest priority than warning, when we have both messages

`warningMessage && !errorMessage `